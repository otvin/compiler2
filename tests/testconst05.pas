program testconst05(output);
{tests character constants vs. string constants.}

const letterq = 'q';

var mychar:char;

procedure writeit(q:char);
begin
	writeln(letterq);
	writeln(q);
end;

begin
	writeln('should be: qqrqq');
	mychar := letterq;
	writeln(mychar);
	writeit('r');
	writeit(letterq);
end.
