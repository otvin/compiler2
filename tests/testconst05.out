should be: qqrqq
q
q
r
q
q
